### PR TITLE
Material3-ify the app and add artifact upload to workflow "Build Debug APK"

### PR DESCRIPTION
**Pull request by user0-tb**

### CODE REVIEW

**Comment by emanuele-f** [comment]
Please describe the changes. The upload step is using a local path

File: app/src/main/res/values/styles.xml
Comment (user0-tb) [inline app/src/main/res/values/styles.xml:1]: Replace Material Components with Material3 Components.

File: .github/workflows/debug-build.yml
Comment (user0-tb) [inline .github/workflows/debug-build.yml:1]: Add artifact upload

Comment (user0-tb) [reply .github/workflows/debug-build.yml:1]: Also change the path to make upload working if repo name will be changed.

**Pull request closed without merging**